store/src/java/com/zimbra/cs/contacts/RelatedContactsResults.java

```diff
@@ -0,0 +1,97 @@
+package com.zimbra.cs.contacts;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.common.base.Joiner;
+import com.google.common.base.Objects;
+
+public class RelatedContactsResults {
```

Comment:
Looking a bit like very similar classes in store and soap.  Is it possible to use just the SOAP ones or even extend them?

Reply:
Yeah those classes pretty much do the same thing. I generally avoid using SOAP classes in store, but in this case it probably makes sense.